rules/library.bzl

```diff
@@ -21,6 +21,8 @@ PrivateHeadersInfo = provider(
     },
 )
 
+GLOBAL_INDEX_STORE_PATH = "bazel-out/rules_ios_global_index_store.indexstore"
```

Comment:
This seems reasonable to put here - given we have in-sight dictating the source of truth of these sourcekit invocations form Bazel

Is it far to say that this path needs to be set by Xcode too or are you over-ridding it.

Reply:
Hep, this is now set in Xcode as well (see `xchammer` PR linked in the PR descr)